packages/rancher-backup/generated-changes/patch/Chart.yaml.patch

```diff
@@ -0,0 +1,16 @@
+--- charts-original/Chart.yaml
++++ charts/Chart.yaml
+@@ -10,6 +10,8 @@
+   catalog.cattle.io/scope: management
+   catalog.cattle.io/type: cluster-tool
+   catalog.cattle.io/ui-component: rancher-backup
++  catalog.cattle.io/kube-version: '>= 1.16.0-0'
++  catalog.cattle.io/upstream-version: '2.1.0'
+ apiVersion: v2
+ appVersion: 2.1.0
+ description: Provides ability to back up and restore the Rancher application running on any Kubernetes cluster
+@@ -19,3 +21,4 @@
+ - infrastructure
+ name: rancher-backup
+ version: 2.1.0
++kubeVersion: '>= 1.16.0-0'
```

Comment:
Note: The annotations and kubeVersion are patched to the chart because it is too late to add them to the upstream chart. Those will be moved to the upstream when the next version is released.